airport-challenge-domain-models.md

```diff
@@ -47,11 +47,9 @@ Initial thoughts
 
 1. Need an airport with a default capacity
 2. Need to be able to change this default capacity.
-3. Airport capacity should only accept integer values.
```

Comment:
I shouldn't have removed this test as I think it would have been good to not allow capacity entries to be strings for example.